tasks/delete_local_filebucket.rb

```diff
@@ -0,0 +1,55 @@
+#!/opt/puppetlabs/puppet/bin/ruby
+# frozen_string_literal: true
+
+require 'json'
+require 'puppet'
+
+params = JSON.parse(STDIN.read)
+force = params['force']
+require_relative File.join(params['_installdir'], 'puppet_agent', 'files', 'rb_task_helper.rb')
+
+module PuppetAgent
+  class DeleteLocalFilebucket
+    include PuppetAgent::RbTaskHelper
+
+    def initialize(force)
+      @force = force
+    end
+
+    def run
+      return error_result(
+        'puppet_agent/no-puppet-bin-error',
+        "Puppet executable '#{puppet_bin}' does not exist",
+      ) unless puppet_bin_present?
+
+      begin
+        FileUtils.remove_dir(clientbucketdir, force)
```

Comment:
It may not worth fixing it, but if the `clientbucketdir` is removed, the puppet command listing local filebuckets may error:
```
scrawny-tumbler:~ # find /opt/puppetlabs/puppet/cache/clientbucket/                 
find: `/opt/puppetlabs/puppet/cache/clientbucket/': No such file or directory
scrawny-tumbler:~ # puppet filebucket -l list
Error: Could not run: File not found
scrawny-tumbler:~ # mkdir /opt/puppetlabs/puppet/cache/clientbucket/
scrawny-tumbler:~ # puppet filebucket -l list
scrawny-tumbler:~ # puppet filebucket backup /etc/passwd --local
scrawny-tumbler:~ #
```

Reply:
Seems that puppet will create the directory again on the next run:
```
Debug: /File[/opt/puppetlabs/puppet/cache/clientbucket]: Adding autorequire relationship with File[/opt/puppetlabs/puppet/cache]
...
Debug: /File[/opt/puppetlabs/puppet/cache/clientbucket]/ensure: created
```

Reply:
Maybe we should just remove the directory contents then?